### ISSUE TITLE
IRC -> Mixer appearing as `filtered`

### ISSUE THREAD

**Issue by gdhgdhgdh** [nerdonthestreet/BeamBridge#1]
Hello - thanks for this software - it nearly worked perfectly first time for me :) Hopefully I did something wrong and you can point me in the right direction!

When I type something in Mixer chat, it appears perfectly in IRC.

However, when I (WebUser19) type the message `qwerty` in IRC, it appears in Mixer chat like this:

![image](https://user-images.githubusercontent.com/5349029/78720126-06019180-791d-11ea-8c45-30157dd8ebb7.png)

At the same time, I see this in the logs:
```
Mixer message received: [WebUser19] <filtered>q<filtered>w<filtered>e<filtered>r<filtered>t<filtered>y<filtered>
```

I'm using `irc.mixxnet.net` as the IRC server. Does this sound like a problem you've seen in the past?

**Comment by jacobgkau** [comment]
Hello there-- this is an interesting one. I wrote this program for a weekly stream who has between 10 and 60 viewers at any given time, and the "mixerBannedWords" setting was a necessity for them-- if we didn't have this restriction, the bot would likely get banned from Mixer for inappropriate/offensive language, sooner or later.

You're seeing that behavior because you didn't specify any words in the mixerBannedWords= line of the config file (and I didn't pre-populate that option in the config because GitHub would likely have an issue with those words as well.) I tested _very_ thoroughly to make sure this setting works, but I never bothered to test it while blank.

I just need to wrap the filter logic inside of an if/then statement so it only runs if mixerBannedWords is not null. In the meantime, you can work around this issue by adding at least one word to mixerBannedWords; it can even be something like "reallylongfakewordnobodywoulduse".

**Comment by gdhgdhgdh** [comment]
Excellent, thank you for the fast and detailed response!

I've popped a random string into `mixerBannedWords` and yep that works like a charm!

My only other question is regarding emoji. Anything from IRC -> Mixer comes through OK since it's just literal ASCII text, but even typing `:)` in Mixer will cause it to be converted to an emoji, and that comes through on IRC as a blank string:

![image](https://user-images.githubusercontent.com/5349029/78764613-b5bb1b80-797e-11ea-83f9-4e798e00f777.png)

```
Mixer message received: 
```

Is it possible to even send to IRC some kind of Slack-style emoji code like `:smile:`?

**Comment by jacobgkau** [comment]
The blank filtering issue was closed by e4a405e1d99524525c2c3d0e453dab5ae9585d5a, which was included in build 057.

The emote issue (which also affected URLs, and cut off the rest of a message after the first emote or URL) was fixed by 66875926ad46d21837e68d0ec84027c14f17db3d, which was included in build 059.
